### DISCUSSION TITLE
Add support for Cloudflared

### DISCUSSION THREAD

**Discussion by Kyescentia** [diced/zipline#473] [Ideas]
Basically a built-in reverse proxy I suppose.. 
https://github.com/cloudflare/cloudflared

Easy reverse proxy setup thru CF

**Comment by diced** [comment]
Why would support for this need to be added? I don't get why you would need to have it built in to Zipline, when cloudflared can do what you want by itself..?

**Reply by Kyescentia** [reply]
It's a built-in reverse proxy without having a separate server running it. Useful for those running on hosts where they do not have root access

**Reply by diced** [reply]
still don't get it...?
